support/windows-client/networking/tcp-ip-connectivity-issues-troubleshooting.md

```diff
@@ -142,3 +142,7 @@ Next, run the command `netsh wfp show state` which generates a *wfpstate.xml* fi
 You can open this file by using Notepad and filter for the ID found in the event logs (for example, 2944008 in the sample event). The result reveals the firewall rule name associated with the Filter ID that is blocking the connection.
 
 :::image type="content" source="media/tcp-ip-connectivity-issues-troubleshooting/wfpstate-file.png" alt-text="Screenshot of the wfpstate xml file which includes the firewall rule name that's associated with the filter ID that's blocking the connection.":::
+
+## See Also
```

Comment:
Should we also add the link directly in the "Packet loss over the network" section of this document?

We could have, right before "Scenario 1", a link with "For more details about diagnosing packet loss issues, see ..."